### ISSUE TITLE
[suggestion] Option to use global variables that are not being saved after restart or wrriten

### ISSUE THREAD

**Issue by UzumakiYT** [SkriptLang/Skript#1596]
I think will be nice to have the ability to use something similar to {_variable} but global not just local

I know about 
```
	# PS: If you don't want some variables to be saved in any database (e.g. variables that contain an %entity% which usually despawn when the server is shut down)
	# you can modify the last database's pattern to not match all variables, e.g. use '(?!x_).*' to match all variables that don't start with '
```
but still, I think is better to have one without depending on config, (also some variables are not being deleted when I used that).

**Comment by TheBentoBox** [comment]
I believe there was a stance taken about not doing this automatically via some prefix because it could be a very serious breaking change if someone happens to already have variables saved with the decided naming convention. It's just too dangerous of a change to make in anything other than a major update like an eventual Skript 3.0. And even then, as you've shown it's easy to set up via config already and this way you can make the global-temp variable have whatever naming pattern you desire.

tl;dr I think this is a no for now but if a dev wants to disagree or correct me please feel free.
